low-level/framework/base/sspl_constants.py

```diff
@@ -151,8 +151,8 @@ class RaidDataConfig(Enum):
     MISMATCH_COUNT_FILE = "/md/mismatch_cnt"
     STATE_COMMAND_RESPONSE = 'idle'
     MISMATCH_COUNT_RESPONSE = '0'
-    RAID_RESULT_DIR = "/tmp"
-    RAID_RESULT_FILE_PATH = "/tmp/result_raid_health_file"
+    RAID_RESULT_DIR = f"{DATA_PATH}raid_integrity/"
+    RAID_RESULT_FILE_PATH = f"{RAID_RESULT_DIR}result_raid_health_file"
```

Comment:
did you see if any error logs coming after this change from raid integrity sensor? Test this sensor and see the results are updated in proper folder & file

Reply:
Sanity result updated in team ras group chat.
https://teams.microsoft.com/l/message/19:95e8d89deb2a4213921cfa821f652e4f@thread.tacv2/1617290058890?tenantId=d466216a-c643-434a-9c2e-057448c17cbe&groupId=7e2a2869-5847-4a65-9402-f4778c7b913e&parentMessageId=1616644644648&teamName=CORTX.All&channelName=EOS-RAS&createdTime=1617290058890

Reply:
I dont see test results it in RAS group chat thread still. Just confirm if RAID integrity was tested, then we should be good for this